src/Symfony/Component/ErrorCatcher/ExceptionHandler.php

```diff
@@ -101,8 +102,12 @@ public function setHandler(callable $handler = null)
      * The latter takes precedence and any output from the former is cancelled,
      * if and only if nothing bad happens in this handling path.
      */
-    public function handle(\Exception $exception)
+    public function handle(\Throwable $exception)
```

Comment:
this cannot be done if the method is not final: we need an annotation on the method in 4.4

Reply:
The whole class is already marked as @final since https://github.com/symfony/symfony/pull/28954, isn't it enough?

Reply:
The deprecated class extends the new one, which is final but DebugClassLoader is not enabled yet when this happens so it doesn't scream... OK... :)

Reply:
It wouldn't trigger the deprecations since they are both "Symfony" classes anyway. 🤔

Reply:
Symfony classes aren't exempted from this rule :)
But let's keep it this way, that's pragmatic.